dev/run-pip-tests

```diff
@@ -105,7 +105,7 @@ for python in "${PYTHON_EXECS[@]}"; do
     sdists=(*.tar.gz)
     if [ ${#sdists[@]} -ne 1 ]; then
       echo "Unexpected number of targets found in dist directory - please cleanup existing sdists first."
-      exit -1
+      exit 255
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.